here-naksha-app-service/src/main/resources/swagger/openapi.yaml

```diff
@@ -524,6 +524,40 @@ paths:
           $ref: '#/components/responses/ErrorResponse504'
         '513':
           $ref: '#/components/responses/ErrorResponse513'
+    delete:
+      tags:
+        - Write Features
+      summary: Delete features in the space
+      description: >
+        This method allows to delete features in the storage associated (directly or via event handler) with the space.
+        It ensures an atomic operation, so either all features will be deleted or none (in case of failure).
```

Comment:
This is special for Delete. While this statement about "atomic" operation is true. Attempting to delete a record which doesn't exist, doesn't fall into "failure" category, as attempting to delete missing feature is not going to result into unexpected outcome. And hence, it should still be considered as success.
In this case, it is possible that `features` and `deleted` array in the response may not have entry for missing feature, but the status code should still be 2xx.
I atleast suggest adding one line in description stating that "attempting to delete missing feature will not be considered as failure", but whether it got really deleted or not can be detected from returned `features` and `deleted` array.